pkg/serviceaccount/jwt.go

```diff
@@ -157,6 +159,7 @@ func (j *jwtTokenAuthenticator) AuthenticateToken(token string) (user.Info, bool
 				if (err.Errors & jwt.ValidationErrorSignatureInvalid) != 0 {
 					// Signature error, perhaps one of the other keys will verify the signature
 					// If not, we want to return this error
+					glog.V(4).Infof("Signature error (key %d): %v", i, err)
```

Comment:
Is this index actually going to be useful in a log message? is the list stored somewhere in a manner where the index matters?